packages/twenty-front/src/modules/object-record/record-table/record-table-body/components/RecordTableRecordGroupsBody.tsx

```diff
@@ -29,21 +34,31 @@ export const RecordTableRecordGroupsBody = () => {
   }
 
   return (
-    <RecordTableBodyRecordGroupDragDropContextProvider>
-      {visibleRecordGroupIds.map((recordGroupId, index) => (
-        <RecordTableRecordGroupBodyContextProvider
-          key={recordGroupId}
-          recordGroupId={recordGroupId}
-        >
-          <RecordGroupContext.Provider value={{ recordGroupId }}>
-            <RecordTableBodyDroppable recordGroupId={recordGroupId}>
-              {index > 0 && <RecordTableRecordGroupEmptyRow />}
-              <RecordTableRecordGroupSection />
-              <RecordTableRecordGroupRows />
-            </RecordTableBodyDroppable>
-          </RecordGroupContext.Provider>
-        </RecordTableRecordGroupBodyContextProvider>
-      ))}
-    </RecordTableBodyRecordGroupDragDropContextProvider>
+    <>
+      <RecordTableBodyRecordGroupDragDropContextProvider>
+        {visibleRecordGroupIds.map((recordGroupId, index) => (
+          <>
+            <RecordTableRecordGroupBodyContextProvider
+              key={recordGroupId}
+              recordGroupId={recordGroupId}
+            >
+              <RecordGroupContext.Provider value={{ recordGroupId }}>
+                <RecordTableBodyDroppable recordGroupId={recordGroupId}>
+                  {index > 0 && <RecordTableRecordGroupEmptyRow />}
+                  <RecordTableRecordGroupSection />
+                  <RecordTableRecordGroupRows />
+                </RecordTableBodyDroppable>
+                {isAggregateQueryEnabled && (
+                  <RecordTableFooter
```

Comment:
Just curious: So RecordTableFooter was introduced only for aggregates? I thought this would also contain the "load more" part due to the name

Reply:
👍